### PR TITLE
Spell check

### PR DESCRIPTION
**Pull request by ACharlesL**
added spell check to input field and changed max char message font size.

### CODE REVIEW

File: group1/comm/templates/comm/index.html [hunk 1]
```diff
@@ -85,7 +85,7 @@ <h4 class="sidebar_description">Quick links</h4>
       <div id="message_bar" class="col-lg-10 col-xs-12 message">
         <span id="charLimitMessage">Don't enter more than 1000 characters</span>
         <div class="input-group input-group-lg">
-          <input id="text" type="text" class="form-control" maxlength="1000" placeholder="Enter a message..." onkeypress="checklength()">
+          <input id="text" type="text" class="form-control" maxlength="1000" placeholder="Enter a message..." onkeypress="checklength()" spellcheck = "true">
```
Comment (ghost) [inline group1/comm/templates/comm/index.html:88]: Pretty nice that they have this built into HTML as an attribute, huh? ;P

Comment (sonmn) [reply group1/comm/templates/comm/index.html:88]: LGTM

File: group1/... [hunk 2]
```diff
@@ -0,0 +1,3 @@
+M	group1/comm/static/comm_tool.css
+M	group1/comm/static/comm_tool.js
+M	group1/comm/templates/comm/index.html
```
Comment (oliver-nowak) [inline group1/...]: @deltin00 looks like you inadvertantly created & added a file called "...". can you remove this from the PR ? then you are G2G.

Comment (ACharlesL) [reply group1/...]: Will do asap, sorry about that.

Comment (ACharlesL) [reply group1/...]: Will do asap, sorry about that.

Comment (ACharlesL) [reply group1/...]: removed "..." file.

**Pull request merged**